code/processes/tickerlogreplay.q

```diff
@@ -345,20 +323,52 @@ merge:{[dir;pt;tablename;mergelimits;h]
  if[gc;.gc.run[]];
  };
 
+// Get all logs that need to be replayed
+getlogs:{
+  if[all null (.replay.tplogfile;.replay.tplogdir);.err.ex[`getlogs;"No log file or directory given."];1];
+  
+  // If in segmented mode, grab meta table and grab log file names from there, return early
+  if[.replay.segmentedmode;
+    metatable:get .Q.dd[hsym first .replay.tplogfile;`stpmeta];
+    :exec distinct logname from metatable where any each tbls in .replay.tablestoreplay;
```

Comment:
Ok this is nice
So what happens if somebody wants to replay a directory of TP log directories in segmented mode? In the non segmented version you can specify a specific log file or a directory containing a set of log files. It's ok if we don't support it, but it should throw an error
Also is this what the meta file is called now? I thought we had changed the name of it to include a datestamp

Reply:
OK good point I can look at that
We didn't add anything to it as there is only meta table per log folder